pymc/distributions/multivariate.py

```diff
@@ -1651,7 +1650,9 @@ def rng_fn(cls, rng, mu, rowchol, colchol, size=None):
         output_shape = size + dist_shape
 
         # Broadcasting all parameters
-        (mu,) = broadcast_dist_samples_to(to_shape=output_shape, samples=[mu], size=size)
+        shapes = [mu.shape, output_shape]
+        sp_shapes = [s[len(size) :] if size == s[: min([len(size), len(s)])] else s for s in shapes]
```

Comment:
Did you test this by hand?
I'd feel better it this line was covered by tests, or refactored to be easier to comprehend (or both)

Reply:
To test that we are getting the correct broadcastable shape, I think the best option is to keep the function `broadcast_dist_samples_shape` and its tests. `shapes_broadcasting` is replaced by `np.broadcast_shapes`. PR updated.
Also, more tests may be needed, using only what I added here was not correct but all tests were passing.